### ISSUE TITLE
Test Connecting Multiple Host Implementation 

### ISSUE THREAD

**Issue by araskachoi** [whiteblock/gossipsub-testing#10]
Will need to verify that two host clients will be able to communicate with one another
- establish a handshake (if applicable)
- verify that packets can be sent from one client to another
- output to stdout and pipe to a file

**Comment by araskachoi** [comment]
@adam-hanna
Do you think you can describe the process briefly (with the exact commands) so that i can easily deploy this on my own?

**Comment by adam-hanna** [comment]
@araskachoi yup! The easiest way to do this is via the subnet command: `$ go run ./cmd/subnet/main.go`

The default will connect 10 hosts together using the whiteblocks peering topology. Make any necessary changes to `configs/subnet/config.json`

See the [readme](https://github.com/agencyenterprise/gossip-host/blob/develop/README.md#usage) for how to manually connect hosts.

**Comment by araskachoi** [comment]
so the issue with using `subnet` would be that it would all be done on one container. We will need to generate multiple containers that are logically separated with their own designated IP.

I was more interested in the command that is required to connect two clients that have been spun up. 

I believe we are using unique node identifiers using the multiaddr format provided by lilbp2p. is there a way to simplify this to just the IP address?

**Comment by adam-hanna** [comment]
yeah, I was building subnet for myself.

You can connect a host to another by passing it the cli flag `-p`. Just give it a comma separated list. This won't work for hosts that are already running, obviously.

I'll add a command to the rpc channel (and implement it on the client) to be able to tell it to connect to another host. In this way, we'll be able to connect, after spinning up.
